WordPress/src/main/java/org/wordpress/android/ui/jetpack/common/adapters/JetpackBackupRestoreAdapter.kt

```diff
@@ -34,8 +36,11 @@ class RestoreDetailsAdapter(
             ViewType.ICON.id -> JetpackIconViewHolder(imageManager, parent)
             ViewType.HEADER.id -> JetpackHeaderViewHolder(uiHelpers, parent)
             ViewType.DESCRIPTION.id -> JetpackDescriptionViewHolder(uiHelpers, parent)
+            ViewType.PROGRESS.id -> JetpackProgressViewHolder(uiHelpers, parent)
             ViewType.ACTION_BUTTON.id -> JetpackButtonViewHolder(uiHelpers, parent)
             ViewType.CHECKBOX.id -> JetpackCheckboxViewHolder(uiHelpers, parent)
+            ViewType.BACKUP_RESTORE_ADDITIONAL_INFORMATION.id ->
```

Comment:
❓ I noticed these two ViewTypes have "BACKUP_RESTORE" prefix, but the others don't have it. Should we remove it or is it intentional?

Reply:
@malinajirka  - Yes, it's intentional. We decided that anything exclusive to a feature should be prefixed. The same pattern in used by scan. Thanks

Reply:
Could you please elaborate a bit more on this. I'm not sure I follow. `JetpackBackupRestoreAdapter` is used by two features (backup and restore)? `BACKUP_RESTORE_ADDITIONAL_INFORMATION` is also used by two features? Which part is exclusive to a feature?